hw/ip/rstmgr/doc/_index.md

```diff
@@ -170,24 +156,20 @@ The reset manager then checks as follows:
 
 ## Hardware Interfaces
 
-{{< incGenFromIpDesc "/hw/top_earlgrey/ip/rstmgr/data/autogen/rstmgr.hjson" "hwcfg" >}}
+### Parameters
+
+The following table lists the instantiation parameters of `rstmgr`.
+
+
+Parameter                   | Default       | Description
+----------------------------|---------------|---------------
+`SecCheck`                  | 1             | Enables reset consistency checks on the leaf reset.  Each check contains a small FSM
+`SecMaxSyncDelay`           | 2             | The default synchronization delay assumptions used in reset consistency checks.  If a design uses a sync cell with more stages of delay, that value should be supplied.
+
 
 ### Signals
 
-Signal                  | Direction | Description
-------------------------|-----------|---------------
-`ast_i.aon_pok`         | `input`   | Input from `ast`.  This signal is the root reset of the design and is used to generate `rst_por_n`.
-`cpu_i.rst_cpu_n`       | `input`   | CPU reset indication.  This informs the reset manager that the processor has reset.
-`cpu_i.ndmreset_req`    | `input`   | Non-debug-module reset request from `rv_dm`.
-`cpu_dump_i`            | `input`   | CPU crash dump state from `rv_core_ibex`.
-`pwr_i.rst_lc_req`      | `input`   | Power manager request to assert the `rst_lc_n` tree.
-`pwr_i.rst_sys_req`     | `input`   | Power manager request to assert the `rst_sys_n` tree.
-`pwr_i.reset_cause`     | `input`   | Power manager indication for why it requested reset, the cause can be low power entry or peripheral issued request.
-`pwr_i.rstreqs`         | `input`   | Peripheral reset requests.
-`pwr_o.rst_lc_src_n`    | `output`  | Current state of `rst_lc_n` tree.
-`pwr_o.rst_sys_src_n`   | `output`  | Current state of `rst_sys_n` tree.
-`resets_ast_o`          | `output`  | Resets used by `ast`.
-`resets_o`              | `output`  | Resets used by the rest of the core domain.
+{{< incGenFromIpDesc "/hw/top_earlgrey/ip/rstmgr/data/autogen/rstmgr.hjson" "hwcfg" >}}
```

Comment:
If you do not want to loose the signal descriptions here, you can transfer them to the `desc` key of the corresponding signals in the hjson file. I.e., the docgen script takes the desc from there and populates the table with that information.

Reply:
o yes, thanks for the reminder. I was actually going to beef up the hjson description but completely forgot.

Reply:
updated.